newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/index.js

```diff
@@ -151,6 +153,27 @@ const BuildSection = ({
     lastModifiedInfoByProjectId,
     setLastModifiedInfoByProjectId,
   ] = React.useState({});
+  const [pageIndex, setPageIndex] = React.useState<number>(0);
+
+  const columnsCount = getItemsColumns(windowWidth);
+
+  const allGameTemplatesCount = React.useMemo(
+    () =>
+      Math.floor(
```

Comment:
maybe this whole calculus should be in the utils.js to keep this component clean